attest/trousers_tpm_linux.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2020 Google Inc.
```

Comment:
Can we keep this in tpm_linux? I don't like having even more OS specific packages.

Reply:
Done